### PR TITLE
upgradeV2

### PR DESCRIPTION
**Pull request by panda-sheep**
#### What type of PR is this?
- [ ] bug
- [X] feature
- [ ] enhancement

#### What does this PR do?
dst_part_file is a json format file, the file name is as follows:

image

#### Checklist:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the corresponding label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
>                                                                 `

### CODE REVIEW

File: src/tools/db-upgrade/DbUpgraderTool.cpp [hunk 1]
```diff
@@ -121,6 +233,32 @@ int main(int argc, char* argv[]) {
...
+  options.skipConfig_ = true;
+  auto metaClient =
+      std::make_unique<nebula::meta::MetaClient>(ioExecutor, std::move(addrs.value()), options);
+  CHECK_NOTNULL(metaClient);
+  if (!metaClient->waitForMetadReady(1)) {
+    LOG(ERROR) << "Meta is not ready: " << FLAGS_upgrade_meta_server;
+    return EXIT_FAILURE;
+  }
+
+  if (!FLAGS_auto_gen_part_schema) {
```
Comment (critical27) [inline src/tools/db-upgrade/DbUpgraderTool.cpp:253]: 👍

File: src/tools/db-upgrade/DbUpgraderTool.cpp [hunk 2]
```diff
@@ -67,28 +74,133 @@ void printHelp() {
...
+    return false;
+  }
+
+  auto spaceVidLenRet = metaClient->getSpaceVidLen(spaceId);
+  if (!spaceVidLenRet.ok()) {
+    LOG(ERROR) << "Get space vid len failed, space id " << spaceId;
+    return false;
+  }
+  auto spaceVidLen = spaceVidLenRet.value();
+  std::unique_ptr<nebula::kvstore::RocksEngine> writeEngine(
```
Comment (critical27) [inline src/tools/db-upgrade/DbUpgraderTool.cpp:122]: I guess the `spaceVidLen` is not actually used? Maybe you could pass a fake one, so we don't need to pass the meta client

Comment (panda-sheep) [reply src/tools/db-upgrade/DbUpgraderTool.cpp:128]: Yes, it seems that it can only be used when the prefix is used.

**Comment by panda-sheep** [comment]
Submitted at https://github.com/vesoft-inc/nebula-ent/pull/381, so close this

**Pull request closed without merging**